traffic_ops/traffic_ops_golang/crconfig/servers.go

```diff
@@ -361,10 +322,10 @@ order by dsr.set_number asc
 	serverDSPatterns := map[tc.CacheName]map[string][]string{}
 	for server, dses := range serverDSNames {
 		for _, dsName := range dses {
-			dsInfList, ok := dsInfs[string(dsName)]
+			dsInfList, ok := dsInfs[dsName]
 			if !ok {
-				if !dsHasTopology[string(dsName)] {
-					log.Warnln("Creating CRConfig: deliveryservice " + string(dsName) + " has no regexes, skipping")
+				if !dsHasTopology[dsName] {
+					log.Warnln("creating CRConfig: deliveryservice " + dsName + " has no regexes, skipping")
```

Comment:
why lowercase this?

Reply:
It's a log line, so capitalization isn't a problem. From [the Golang wiki's Go Code Review Comments section](https://github.com/golang/go/wiki/CodeReviewComments#error-strings):

> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use `fmt.Errorf("something bad")` not `fmt.Errorf("Something bad")`, so that `log.Printf("Reading %s: %v", filename, err)` formats without a spurious capital letter mid-message. **This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.**

(emphasis mine)

Reply:
I don't really care; it's unnecessary, IMO, but not harmful to change it. Just thought I'd bring it up. Feel free to leave it as-is.

Reply:
Oh, good to know. I'll keep that in mind for next time 👍